src/main/java/service/DataWriter.java

```diff
@@ -0,0 +1,8 @@
+package service;
+
+import java.util.Map;
+import model.Fruit;
+
+public interface DataWriter {
+    void writeToFile(Map<Fruit, Integer> fruitReport, String fileTo);
```

Comment:
fileTo is a file or a path to file?

Reply:
for clarification - renamed it to the `fileName`